mantidproject/mantid
# Indirect Moments Crash

**Description of work.**
This PR adds prevents a vector subscript out of range error which can be triggered using the Moments tab of Data Reduction.

**To test:**
1.`Interfaces`->`Indirect`->`Data Reduction`->`Moments` tab
2. Load the data below
3. Click `Run`
An error should not occur.

**Data Files**
[iris26176_graphite002_sqw.zip](https://github.com/mantidproject/mantid/files/3003813/iris26176_graphite002_sqw.zip)

Fixes #25335

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
